project-code/app/com/edulify/modules/geolocation/GeolocationPlugin.java

```diff
@@ -1,39 +0,0 @@
-package com.edulify.modules.geolocation;
-
-import com.edulify.modules.geolocation.providers.FreegeoipProvider;
-import play.Application;
-
-public class GeolocationPlugin extends play.Plugin {
```

Comment:
Yeah, but it'll lead to situation when we'll have two similar component infrastructures one at plugin and one at DI container + the plugin infrastructure will be singleton. Should I really do It?